test/api/unit/services/Domain.test.js

```diff
@@ -1,8 +1,10 @@
-const { expect } = require('chai');
+const { expect, assert } = require('chai');
```

Comment:
```suggestion
const { expect } = require('chai');
```

Reply:
I thought I already took care of this. Perhaps it showed up again.